10/alpine/Dockerfile

```diff
@@ -40,19 +39,13 @@ RUN addgroup -g 1000 node \
     && grep " node-v$NODE_VERSION.tar.xz\$" SHASUMS256.txt | sha256sum -c - \
     && tar -xf "node-v$NODE_VERSION.tar.xz" \
     && cd "node-v$NODE_VERSION" \
-    && ln -s /usr/bin/ccache /usr/local/bin/cc \
-    && ln -s /usr/bin/ccache /usr/local/bin/gcc \
-    && ln -s /usr/bin/ccache /usr/local/bin/g++ \
     && HOST_IP="$(ip route | awk '/default/ { print $3 }')" \
     && nc -v "$HOST_IP" 5678 | tar -xz -C / || true \
```

Comment:
These are the most problematic bits (`nc` to the host machine IP is really hacky), so I'd recommend including them in this revert.